bundles/org.openhab.binding.nanoleaf/src/main/java/org/openhab/binding/nanoleaf/internal/NanoleafBindingConstants.java

```diff
@@ -71,9 +71,12 @@
     public static final String API_MIN_FW_VER_LIGHTPANELS = "1.5.0";
     public static final String API_MIN_FW_VER_CANVAS = "1.1.0";
     public static final String MODEL_ID_LIGHTPANELS = "NL22";
-    public static final String MODEL_ID_CANVAS = "NL29";
+
+    public static final String[] MODELS_WITH_TOUCHSUPPORT = { "NL29", "NL42" };
```

Comment:
Since you use it as list why not create a list here:
```suggestion
    public static final List<String> MODELS_WITH_TOUCHSUPPORT = Arrays.asList("NL29", "NL42");
```

Reply:
definitely more elegant. thx.